applications/crossbar/doc/allotments.md

```diff
@@ -0,0 +1,269 @@
+/*
+Section: Crossbar
+Title: Allotments
+Language: en-US
+*/
```

Comment:
Can you add `Version: 3.22` so the reader knows what version this doc applies to? Big thanks for doing the doc!